app/models/agents/data_output_agent.rb

```diff
@@ -156,6 +157,12 @@ def feed_description
       interpolated['template']['description'].presence || "A feed of Events received by the '#{name}' Huginn Agent"
     end
 
+    def xml_namespace
+      if (interpolated['itunes_compatible'] == "true")
```

Comment:
Use our `boolify` helper to handle `true` as well.

`if (boolify(interpolated['itunes_compatible']))`